ignite/templates/module/placeholders.go

```diff
@@ -27,4 +27,8 @@ const (
 	PlaceholderTypesGenesisValidField = "// this line is used by starport scaffolding # types/genesis/validField"
 	PlaceholderGenesisTestState       = "// this line is used by starport scaffolding # genesis/test/state"
 	PlaceholderGenesisTestAssert      = "// this line is used by starport scaffolding # genesis/test/assert"
+
+	// Migrations
+	PlaceholderMigrationsRegister = "// this line is used by starport scaffolding # migrations/register"
+	PlaceholderMigrationsMigrator = "// this line is used by starport scaffolding # migrations/migrator"
```

Comment:
Not particular to this update but generally, should the comments say "Ignite scaffolding"?